package/Dockerfile.executor

```diff
@@ -10,7 +10,10 @@ RUN curl -sLf https://releases.hashicorp.com/terraform/1.0.0/terraform_1.0.0_lin
 
 COPY terraform-executor /usr/bin/
 
-RUN mkdir -p /root/module
-WORKDIR /root/module
+RUN adduser -D agent
```

Comment:
some systems like OpenShift prefer using specific uid=1001 for non-root user, but we can always fix it later.